ui/Makefile

```diff
@@ -54,22 +56,34 @@ GOBINDATA_SOURCES       := $(BOWER_COMPONENTS)/d3/d3.min.js \
                            $(BOWER_COMPONENTS)/mithriljs/mithril.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.min.css \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.min.js \
+                           $(BOWER_COMPONENTS)/moment/min/moment.min.js \
+                           $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.min.js \
                            $(GOBINDATA_FONTS)
 GOBINDATA_DEBUG_SOURCES := $(BOWER_COMPONENTS)/d3/d3.js \
                            $(BOWER_COMPONENTS)/lodash/lodash.js \
                            $(BOWER_COMPONENTS)/mithriljs/mithril.js \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.css \
                            $(BOWER_COMPONENTS)/nvd3/build/nv.d3.js \
+                           $(BOWER_COMPONENTS)/moment/moment.js \
+                           $(BOWER_COMPONENTS)/moment-timezone/builds/moment-timezone-with-data.js \
```

Comment:
I started doing this, but moment make the logic a lot simpler for now. I'll open an issue to move all timezone logic to the server.